sdk/iot/.gitignore

```diff
@@ -2,3 +2,4 @@
 !*common.config.json
 !*common.test.assets.config.json
 *.etl
+*launchSettings.json
```

Comment:
shouldn't this already be covered by the repo root .gitignore file? If not, I'd say it should.

It is not IoT-specific, so we should push things like this to the root.

Reply:
The root gitignore only has launch.json ...
I will update it to include it in there as well. We also have this in our adt gitignore file.